apps/phone/src/apps/contacts/components/List/SearchContacts.tsx

```diff
@@ -19,10 +19,10 @@ export const SearchContacts: React.FC = () => {
 
   return (
     <div className="w-full py-2 px-4">
-      <div className="flex items-center justify-start space-x-2 rounded-md bg-gray-200 px-2 dark:bg-neutral-800">
+      <div className="flex items-center justify-start bg-neutral-200 dark:bg-neutral-800 rounded-md px-2 space-x-2">
```

Comment:
why tailwind ... ACK REJECTED



jk LGTM

Reply:
Yes, that's what happens when you revert broken commits 😅